packages/react/src/AnchoredOverlay/AnchoredOverlay.features.stories.tsx

```diff
@@ -70,8 +102,8 @@ export const CustomAnchorId = () => {
       renderAnchor={props => <Button {...props}>Button</Button>}
       anchorId="my-custom-anchor-id"
     >
-      <Box width="100%" height="100%" backgroundColor="thistle" display="flex" flexDirection="column">
-        <img src={`//placekitten.com/200/300`} alt="kitten" />
+      <Box width="100%" height="100%" display="flex" flexDirection="column">
```

Comment:
One more small non-blocking thing:
When using `Box` for styling, we prefer people use the `sx` prop instead of `styled-system` props.

Like this...
```suggestion
      <Box sx={{width: '100%', height: '100%', display: 'flex', flexDirection: 'column' }}>
```

Sorry I missed this before 😢

Reply:
no worries at al, thanks for all the input, really appreciate it!